Editor/CoreAPI/StateManager.cs

```diff
@@ -49,6 +49,7 @@ public string BucketName
             {
                 _selectedProfile.BucketName = value;
                 SaveProfiles();
+                CoreApi.PutSetting(SettingsKeys.CurrentBucketName, value);
```

Comment:
DeploymentSettings is already pulling the value directly from StateManager (https://github.com/aws/amazon-gamelift-plugin-unity/blob/feature/anywhere/Editor/Deployment/DeploymentSettings.cs#L190), so I'm not following why this is needed as opposed to providing the value as an argument to the StartDeployment call: https://github.com/aws/amazon-gamelift-plugin-unity/blob/feature/anywhere/Editor/Deployment/DeploymentSettings.cs#L398

Reply:
We could add `Region` and `BucketName` as a parameter to `DeployerBase.StartDeployment` and then add them to `RequestFactory.CreateRequest` if that is the direction we want to go

Reply:
Yeah, that was what I was thinking